frontend/public/components/_nav.scss

```diff
@@ -45,8 +45,10 @@ $active-border: 3px;
 .navigation-container {
   flex: 1 1 auto;
   overflow-x: hidden;
-  overflow-y: auto;
   -webkit-overflow-scrolling: touch;
+  &:hover {
```

Comment:
This is ok on touch devices?

Reply:
Yes,  ios defaults to using the iphones scroll handle. On android the scroll handle is shown when   `.navigation-container` detects touch.

Reply:
it doesn't.
I'll separate it out.